autometa/common/external/diamond.py

```diff
@@ -582,12 +555,14 @@ def main():
         type=int,
     )
     parser.add_argument(
-        "--nproc", help="number of processors to use", default=mp.cpu_count(), type=int
+        "--cpus", help="number of processors to use", default=mp.cpu_count(), type=int
+    )
+    parser.add_argument(
+        "--tmpdir", help="</path/to/tmp/directory>", default=tempfile.mkdtemp()
```

Comment:
Depending on how you handle the comments about the `tmpdir` above, you may be able to remove this.